### DISCUSSION TITLE
Welcome to cohort7_structure Discussions!

### DISCUSSION THREAD

**Discussion by tejuafonja** [AISaturdaysLagos/cohort_structure#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
My name is Olusola Joshua,a data analyst,an undergraduate of the department of systems engineering,Unilag,also I am a back-end dev(python{django}).
I intend to use this community to develop my skill in AI and ML..

**Reply by tejuafonja** [reply]
Welcome, Joshua, and thanks for using the discussion board😉

**Comment by chisomloius** [comment]
Hello, my name is Chisom. It is so nice to be here and I hope to learn, build and have mad fun. OMo....

**Comment by HamplusTech** [comment]
Hello! I have been following at the back. I am JohnPaul and a lover of knowledge!

**Reply by OyinAwosan** [reply]
Hi JohnPaul, so nice to meet you. Do you use Slack?

**Comment by RiddyRichie** [comment]
Good evening, I really thought i can finish the task 3 within three days but nah it wasn't that easy that i thought but i found somehow interesting to do but i encountered something which i don't whether it is an error or not.
![Screenshot (422)](https://user-images.githubusercontent.com/89553525/140582298-c3f0a57d-e453-4b94-aa87-24896d756a81.png)
